fetch_umbrella_build_and_test.sh

```diff
@@ -27,4 +27,14 @@ cmake .. -DGUI=0 -DCMAKE_BUILD_TYPE=$TRAVIS_BUILD_TYPE $OPTIONS
 make lllc solc soljson soltest
 
 
-./solidity/test/soltest
+# Test runs disabled for macos for now,
+# we need to find a way to install eth.
```

Comment:
Yes something like that. The biggest problem, though, is that we do not have a version for macos that provides the required features that are part of the ubuntu nightly snapshot.